src/containers/ServiceAccounts/ServiceAccounts.test.js

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { fireEvent } from 'react-testing-library';
+import { Provider } from 'react-redux';
+import { Route } from 'react-router-dom';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import ServiceAccounts from '.';
+import { renderWithRouter } from '../../utils/test';
+import * as API from '../../api';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+const byNamespace = {
+  default: {
+    'foo-service-account': '1234567890'
+  }
+};
+
+const byId = {
+  '1234567890': {
+    metadata: {
+      name: 'foo-service-account',
+      namespace: 'default',
+      uid: '1234567890'
+    }
+  }
+};
+
+const namespaces = {
+  byName: {
+    default: {
+      metadata: {
+        name: 'default'
+      }
+    }
+  },
+  errorMessage: null,
+  isFetching: false,
+  selected: '*'
+};
+
+it('ServiceAccounts renders with no bindings', () => {
+  jest.spyOn(API, 'getServiceAccounts').mockImplementation(() => []);
+
+  const store = mockStore({
+    serviceAccounts: {
+      byId: {},
+      byNamespace: {},
+      isFetching: false,
+      errorMessage: null
+    },
+    namespaces,
+    notifications: {}
+  });
+
+  const { getByText } = renderWithRouter(
+    <Provider store={store}>
+      <Route
+        path="/serviceaccounts"
```

Comment:
should use the router utils, e.g. `paths.serviceAccounts.all()` instead of hardcoding `/serviceaccounts` here